src/Spago/PackageSet.hs

```diff
@@ -1,5 +1,7 @@
 module Spago.PackageSet
-  ( upgradePackageSet
+  ( useLatestRelease
+  , useSpecificRelease
+  , updatePackageSetVersion
```

Comment:
We're not using these anywhere so they shouldn't really be exported:

```suggestion
  ( updatePackageSetVersion
```

Reply:
Done.